src/test/fixtures/sample-config- fail-publish-document.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
There is a space in the filename, dont think that is intentional

Reply:
ok, yup must be a typo, i'll update it.